internal/pkg/api/profile/list.go

```diff
@@ -39,6 +41,22 @@ func ProfileList(ShowOpt *wwapiv1.GetProfileList) (profileList wwapiv1.ProfileLi
 					fmt.Sprintf("%s:=:%s:=:%s", p.Id.Print(), f.Field, f.Value))
 			}
 		}
+	} else if ShowOpt.ShowYaml {
+		profileMap := make(map[string]node.NodeInfo)
+		for _, profile := range profiles {
+			profileMap[profile.Id.Get()] = profile
+		}
+
+		buf, _ := yaml.Marshal(profileMap)
+		profileList.Output = append(profileList.Output, string(buf))
+	} else if ShowOpt.ShowJson {
+		profileMap := make(map[string]node.NodeInfo)
+		for _, profile := range profiles {
+			profileMap[profile.Id.Get()] = profile
+		}
+
+		buf, _ := json.Marshal(profileMap)
```

Comment:
Sorry to jump again, but could we use 
```
			buf, _ = json.MarshalIndent(nodeResp, "", "  ")
```
instead. This is still valid json, but more human readable?
Or do you have any objections?

Reply:
https://github.com/warewulf/warewulf/pull/1053/commits/5956f2e9a1871c923eab2db830ff7dba14c3b83a
thanks for the comment, it's updated in this commit.